test/test_cuda.py

```diff
@@ -442,6 +442,10 @@ def test_out_of_memory_retry(self):
         IS_JETSON, "oom reporting has issues on jetson igx due to partial nvml support"
     )
     def test_set_per_process_memory_fraction(self):
+        if torch.version.hip:
```

Comment:
Please add GPU arch check and put comments with explaination

Reply:
Added the arch check